src/platform-includes/configuration/config-intro/python.mdx

```diff
@@ -9,11 +9,7 @@ sentry_sdk.init(
     dsn="___PUBLIC_DSN___",
     max_breadcrumbs=50,
     debug=True,
-
-    # Set traces_sample_rate to 1.0 to capture 100%
-    # of transactions for performance monitoring.
-    # We recommend adjusting this value in production,
-    traces_sample_rate=1.0,
+    enable_tracing=True,
```

Comment:
Does this set a default sample rate of 100%? 
Should we consider leaving in a similar comment that warns users?

Reply:
@shanamatthews  Yes, `enable_tracing=True` is equal to `traces_sample_rate=1.0`. But warning users is no longer necessary (at least in this SDK), they can also set it to 1.0 in production and the SDK will take care of tuning down in case of heavy load. Next quarter we're working on backpressure management in the Java and Ruby SDK, so that we can deliver the same (simpler and safer) experience there.